packages/core/src/types/ICamera.ts

```diff
@@ -28,6 +29,12 @@ interface ICamera {
   flipHorizontal?: boolean;
   /** flip Vertical */
   flipVertical?: boolean;
+  /** physical scale */
```

Comment:
if we have no intention for our end users to ever use this, we should leave it out